### DISCUSSION TITLE
Improving the way we surface website settings

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1452] [Anything Else]
Sometimes people new to Plausible get confused between account settings and site settings. To help out discover the site settings easier, perhaps we can make the cog on the /sites landing page be displayed at all times rather than only when hover over? And then perhaps somehow visually show that this is a different button from the main button (different background color or a border or something?)

**Comment by superterrific** [comment]
I'm new to Plausible and had exactly that confusion, it took me a few days to find the site settings. It looks like the settings cog icon is displayed persistently on smaller screens, but only on hover on larger screens (which is where I primarily use Plausible at the moment). 

I would have figured it out much sooner if the cog was persistent on the My Sites page!

**Reply by ukutaht** [reply]
Thanks @superterrific! We'll definitely make this change :)

**Comment by metmarkosaric** [comment]
The cog icon on /sites is now visible at all times

**Reply by superterrific** [reply]
Thanks @metmarkosaric & @ukutaht! Enjoying Plausible so far btw :)

**Reply by metmarkosaric** [reply]
great to hear that, thanks!
